prog/postgres/postgres_server_nexus.rb

```diff
@@ -166,10 +166,41 @@ def before_run
 CONFIG
     vm.sshable.cmd("sudo -u prometheus tee /home/prometheus/web-config.yml > /dev/null", stdin: web_config)
 
-    vm.sshable.cmd("sudo systemctl enable --now postgres_exporter")
-    vm.sshable.cmd("sudo systemctl enable --now node_exporter")
-    vm.sshable.cmd("sudo systemctl enable --now prometheus")
-    hop_configure
+    metric_destinations = postgres_server.resource.metric_destinations.map {
```

Comment:
What happens if prometheus can't connect to the remote destinations? Does it have a timeout?

Reply:
Yes, it has 30 second timeout by [default](https://prometheus.io/docs/prometheus/latest/configuration/configuration/#remote_write).